### ISSUE TITLE
Implementing `touch` method to support `resave: false`?

### ISSUE THREAD

**Issue by martianyi** [origin1tech/connect-sequelize#12]
According to the express-session documentation on resave: https://github.com/expressjs/session#resave, It would be nice to implement a `touch` method to support `resave` option.

**Comment by origin1tech** [comment]
Agreed might you create a PR for that? I honestly don't have the time at the moment. Should be rather trivial.
